cwl/tools/format_annoSV.cwl

```diff
@@ -13,10 +13,10 @@ arguments:
   - position: 0
     shellQuote: false
     valueFrom: >-
-      -sv $(inputs.input_annoSV.path) -participant_id $(inputs.participant_id) -biospecimen_id_normal $(inputs.biospecimen_id_normal) -biospecimen_id_tumor $(inputs.biospecimen_id_tumor) |gzip >$(inputs.input_annoSV.nameroot).$(inputs.biospecimen_id_tumor).gz
+      -sv $(inputs.input_annoSV[0].path) -participant_id $(inputs.participant_id) -biospecimen_id_normal $(inputs.biospecimen_id_normal) -biospecimen_id_tumor $(inputs.biospecimen_id_tumor) |gzip >$(inputs.input_annoSV[0].nameroot).$(inputs.biospecimen_id_tumor).gz
 
 inputs:
-  input_annoSV: File
+  input_annoSV: File[]
```

Comment:
What's going on here? Why are we getting an array but only using the first file in the array in the command?

Reply:
I see. Using the conditional workflows should alleviate this issue.